XenAdmin/Wizards/ExportWizard/ExportSelectVMsPage.cs

```diff
@@ -301,7 +302,10 @@ private long GetFreeSpace(string drivename)
 		private void EnableButtons()
 		{
 			var count = VMsToExport.Count;
-			m_tlpWarning.Visible = ExportAsXva && count > 1;
+            m_tlpInfo.Visible = ExportAsXva && count > 1;
+            _tlpWarning.Visible = !Helpers.FeatureForbidden(Connection, Host.RestrictVtpm) &&

```

Comment:
Indentation

Reply:
It's due to mixed tabs and spaces in this file. I'll normalize them.